src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTagRecordsAndCompactCtors.java

```diff
@@ -1,41 +1,49 @@
 /*
 WriteTag
 tag = @incomplete
-tagFormat = \S
-tagSeverity = (default)info
-tokens = INTERFACE_DEF, CLASS_DEF, ENUM_DEF, ANNOTATION_DEF, RECORD_DEF, COMPACT_CTOR_DEF
+tagSeverity = error
+tagFormat = \\S
+tokens = INTERFACE_DEF, CLASS_DEF, ENUM_DEF, ANNOTATION_DEF, RECORD_DEF, COMPACT_CTOR_DEF, CTOR_DEF
 
 
 */
 
 //non-compiled with javac: Compilable with Java14
 package com.puppycrawl.tools.checkstyle.checks.javadoc.writetag;
 
-public class InputWriteTagRecordsAndCompactCtors {
+// violation below , 'Type Javadoc comment is missing @incomplete tag'
+public class InputWriteTagRecordsAndCompactCtors { // violation
```

Comment:
Here I have written ```violation below``` at line 14 and then again mentioned ``` // violation``` on line 15. If I don't mention ```// violation``` on line 15 then it gives error. If I mention ```// violation , 'Type Javadoc comment is missing @incomplete tag' ``` on line 15 then also it gives error.

Reply:
Please debug a problem.

Reply:
@romani refer my comment https://github.com/checkstyle/checkstyle/pull/12288#issuecomment-1295752000